nss-day-cohort-41/tabloid-fullstack-first-amendment-dancers-andy
# Ak user profile list and details

# Description
Can now See UserProfiles List and UserProfiles Details
Fixes # (23, 24)
## Type of change
Please delete options that are not relevant.


# Testing Instructions
``Git fetch origin ak-user-profile``
``Git checkout origin ak-user-profile``

From navbar select UserProfiles after logging in. Can view all current profiles:shoud see full name, displayname and usertype.
if you select the link in UserName it will take you to Details where you can see full name, avatar, displayname, email, creationdate (listed as MM/DD/YYY) and userprofiletype. 
No changes can be made to profiles and anyone can view
go back button should take you back to full list

category edit should allow user to not edit and still save

# Checklist:
